terraform/notification.canada.ca-zone.tf

```diff
@@ -29,14 +29,26 @@ resource "aws_route53_record" "notification-canada-ca-ALIAS" {
   }
 }
 
-resource "aws_route53_record" "api-notification-canada-ca-A" {
+resource "aws_route53_record" "api-notification-canada-ca-A-k8s" {
   zone_id = aws_route53_zone.notification-canada-ca-public.zone_id
   name    = "api.notification.canada.ca"
   type    = "CNAME"
-  records = [
-    local.notification_alb
-  ]
-  ttl = "300"
+  records = [local.notification_alb]
+  ttl     = "300"
+  weighted_routing_policy {
+    weight = 100
```

Comment:
Can you switch this to 255 so that it gets all traffic:
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-values-weighted.html#rrsets-values-weighted-weight